### ISSUE TITLE
Required strings are nullable

### ISSUE THREAD

**Issue by DaGrisa** [domaindrivendev/Swashbuckle.AspNetCore#1571]
Hello,

when marking string type attributes as "required", they are "nullable":
```
[Required]
public string Category { get; set; }

[Required]
public string Id { get; set; }
```

![image](https://user-images.githubusercontent.com/11348169/76759777-a7694d80-678c-11ea-9d8e-6817a9cb2e1b.png)

How am I able to avoid this?

Thank you in advance,
Daniel

**Comment by domaindrivendev** [comment]
Which version of SB. This should be resolved from as of v5.1.0

**Comment by DaGrisa** [comment]
We just updated from 5.0.0 to 5.1.0 before.

**Comment by domaindrivendev** [comment]
You're correct this isn't working 5.1.0 and it _should_ be - bumping up the priority to get it fixed ASAP

**Comment by domaindrivendev** [comment]
Fixed by 032ff572e46734cefdbbd487ffb2634c8cedb1fd

**Comment by TimothyByrd** [comment]
Until 5.2.0 comes out, I've made a document filter to work around this.

https://github.com/TimothyByrd/swashbuckle_stuff
